deployments/container/reconfigure-mig.sh

```diff
@@ -349,6 +349,77 @@ EOF
 	return 0
 }
 
+function get_driver_library_path() {
+	local search_paths=("/usr/lib64"
+                      "/usr/lib/x86_64-linux-gnu"
+                      "/usr/lib/aarch64-linux-gnu"
+                      "/lib64"
+                      "/lib/x86_64-linux-gnu"
+                      "/lib/aarch64-linux-gnu"
+                      )
+	for search_path in "${search_paths[@]}"
+	do
+		path=$(find "/host/${DRIVER_ROOT}${search_path}" -name "libnvidia-ml.so.1" 2>/dev/null | head -n 1)
+		if [ ! -z $path ]; then
```

Comment:
Question: Why `! -z` and not `-n`?

Reply:
No particular reason. I have left this as is, since this code is removed in my follow-up: https://github.com/NVIDIA/mig-parted/pull/85